src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardDetails.tsx

```diff
@@ -51,6 +51,16 @@ const IfoCardDetails: React.FC<IfoCardDetailsProps> = ({ poolId, ifo, publicIfoD
   const totalLPCommittedInUSD = currencyPriceInUSD.times(totalLPCommitted)
   const totalCommitted = `~$${formatNumber(totalLPCommittedInUSD.toNumber(), 0, 0)} (${totalCommittedPercent}%)`
 
+  const PricePerTokenWithFeeToOriginalRaio = poolCharacteristic.sumTaxesOverflow
```

Comment:
variable names should start with lower case letter
same for PricePerTokenWithFee